src/styles/variables.scss

```diff
@@ -0,0 +1,12 @@
+$day-size: 100px;
+$border-day-gap: 1px;
+$text-color: #000;
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thur: 3,
```

Comment:
There is a typo in the day abbreviation 'thur'. It should be 'thu' to correctly represent Thursday.